feat: add more gsoc 2024 projects (#7302)
jenkins-infra/jenkins.io: content/_data/authors/TheMeinerLP.adoc

---
name: "Phillipp Glanz"
github: TheMeinerLP
twitter: themeinerlp
---

> // descriptive text comes here

Reviewer: ```suggestion

```
Remove unnecessary text

> Phillipp is the owner of link:https://github.com/OneLiteFeatherNET[@OneLiteFeatherNET] & an admin of projects. He is contributing to the GSoC 2024 "Improve Maintainability for the Repository Permission Updater" project to automate the management of GitHub permissions to improve maintainability for the Repository Permission Updater for Jenkinsci.

Reviewer: ```suggestion
Phillipp is the owner of link:https://github.com/OneLiteFeatherNET[@OneLiteFeatherNET] and a project admin.
He is contributing to the GSoC 2024 "Improve Maintainability for the Repository Permission Updater" project to automate the management of GitHub permissions to improve maintainability for the Repository Permission Updater for Jenkinsci.
```
Simplify the first line
